### PR TITLE
[242] Allow hiding profiles from entire trusts

### PR DESCRIPTION
**Pull request by elliotcm**
Screenshot 2023-03-27 at 12 09 08

Screenshot 2023-03-27 at 12 22 44

Screenshot 2023-03-27 at 13 31 36

### CODE REVIEW

**Comment by elliotcm** [comment]
[Update Hide from profile process with "Delete trust" text](https://trello.com/c/289Yh3xb/255-update-hide-from-profile-process-with-delete-trust-text)

File: config/locales/jobseekers.yml [hunk 1]
```diff
@@ -534,13 +534,21 @@ en:
...
         delete:
-          delete_school: Delete school
+          delete_school: Delete organisation
           page_title: Confirm that you no longer want to hide your profile from %{name}, %{address}
         review:
           page_title: Hide profile from schools or trusts
         schools:
           add_another: Do you want to add another school or trust?
+          already_hidden: You are already hidden from %{name}
+          hidden_from_trust_and_schools: Your profile will be hidden from all schools within this Trust.
```
Comment (josephhull676) [inline config/locales/jobseekers.yml:551]: Bit nitpicky, but I just spotted it after scanning over these translations... Are we sure we want to capitalise Trust?

File: spec/system/jobseekers_can_manage_a_profile_spec.rb [hunk 2]
```diff
@@ -473,6 +473,123 @@
...
+
+      it "does not allow the jobseeker to hide themselves from the school again" do
+        visit jobseekers_profile_path
+        click_on I18n.t("jobseekers.profiles.show.set_up_profile_visibility")
+        choose "Yes", visible: false
+        click_on I18n.t("buttons.save_and_continue")
+
+        field = find_field("Name of school or trust")
+        field.fill_in(with: forbidden_organisation.name)
+        field.native.send_keys(:tab)
```
Comment (josephhull676) [inline spec/system/jobseekers_can_manage_a_profile_spec.rb:493]: What's the purpose of this in this test?

**Review by josephhull676 — APPROVED** [review]

File: spec/system/jobseekers_can_manage_a_profile_spec.rb (hunk 2 above)
Comment (elliotcm) [reply spec/system/jobseekers_can_manage_a_profile_spec.rb:493]: It triggers the JS autocomplete search thing, to actually select the suggested result.

**Pull request merged**